### ISSUE TITLE
custom view suport

### ISSUE THREAD

**Issue by Gavin-Whitesitt** [tableau/tabcmd#308]
From the release notes:

"Known gaps from classic tabcmd: custom views are not supported in any operations"

I work at an org that has MFA enabled, so this is my only option for automating a lot of our workflow. My team relies heavily on custom views, are there plans/timeline to bring this functionality to this tool? We use Tableau Server Version: 2023.3.1

**Comment by jacalata** [comment]
Yes, adding this across various operations is the next set of features. Are there any specific actions you would prioritize?

**Comment by Gavin-Whitesitt** [comment]
The ability to export a custom view to either PDF/crosstab/PowerPoint with (with refresh on, not from cache). This is 80% of my use case.  It would also be great to be able to add additional filters in the statement itself. This is the other 20% of my usecase. Thanks!

**Comment by jacalata** [comment]
@Gavin-Whitesitt Exporting a custom view is now available in v2.0.14. I think this will meet all your use-cases, but please let me know if it doesn't.

**Comment by Matuvu38** [comment]
Hello @jacalata  it seems exporting custom view does not work. When I am trying to export custom view, the script actually export the original view.

**Comment by jacalata** [comment]
Hi @Matuvu38 , can you add some more detail? I haven't seen that happening myself.

**Comment by Matuvu38** [comment]
Hello @jacalata, sorry for the delay. The custom view actually works. However I could have empty exports when the report is bit long to upload. It seems the export is triggered before the data loading is finished. And I did not have this problem with tabcmd 1.0
For information the command I am using  for '.csv' for exemple is : 
tabcmd export "Worboook_Link" --csv -f "File_Link.csv"

**Comment by jacalata** [comment]
Thanks  - That sounds like an issue we have seen and needs a fix on the backend - https://github.com/tableau/tabcmd/issues/310. I'm going to close this issue for custom views, and will post in 310 with any update on the blank export pages.
